packages/identity-app/src/SavedAccounts.tsx

```diff
@@ -2,19 +2,30 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { ApiContext } from '@polkadot/ui-api';
 import { AddressSummary, MarginTop, Stacked, WalletCard, WithSpace } from '@polkadot/ui-components';
+import accountObservable from '@polkadot/ui-keyring/observable/accounts';
+import { SingleAddress, SubjectInfo } from '@polkadot/ui-keyring/observable/types';
 import React from 'react';
 import { Link } from 'react-router-dom';
 
 type Props = {
   basePath: string
 };
 
-export class SavedAccounts extends React.PureComponent<Props> {
-  static contextType = ApiContext;
+type State = {
+  allAccounts?: SubjectInfo
+};
+
+export class SavedAccounts extends React.PureComponent<Props, State> {
+  state: State = {};
 
-  context!: React.ContextType<typeof ApiContext>; // http://bit.ly/typescript-and-react-context
+  componentDidMount () {
+    accountObservable.subject.subscribe(accounts => {
```

Comment:
I believe unsubscribe is missing in unmount.

Reply:
@jacogr i added unsubscribe on component unmount, but while keeping the subscriptions in state I couldn't find the Subscription type anywhere in `@types/rx` - is this actually somewhere?

Reply:
It is an Rx observable that gets returned there, I don't think we created a specific type for it since it is just an `Observable<type>`. @amaurymartiny am I correct?

Reply:
See https://github.com/paritytech/substrate-light-ui/pull/49#discussion_r255891715